django/src/games/templates/pong.html

```diff
@@ -1,278 +1,61 @@
 {% extends "./extends/base.html" %}
 
 {% block content %}
-<div class="h-100 d-flex flex-column justify-content-center align-items-center">
+<div class="container">
 	<div id="main" game-menu>
-		<button menu-id="local">Local Play</button>
-		<button menu-id="online">Online Play</button>
+		<button class="btn btn-primary" menu-id="local">Local Play</button>
+		<button class="btn btn-danger" menu-id="online">Online Play</button>
 	</div>
 
 	<div id="local" game-menu>
-		<button join-mode="local">Play with a Friend</button>
-		<button join-mode="ai">Play with AI</button>
+		<a class="btn btn-primary text-decoration-none" data-route href="/pong/local">Play with a Friend</a>
+		<a class="btn btn-danger text-decoration-none" data-route href="/pong/ai">Play with an AI</a>
 	</div>
 
 	<div id="online" game-menu>
-		<button join-mode="online">Create a Room</button>
-		<button menu-id="join">Join a Room</button>
-		<button>Join a Tournament</button>
+		<a class="btn btn-primary text-decoration-none" data-route href="/pong/online">Create a Room</a>
+		<button class="btn btn-danger text-decoration-none" menu-id="join">Join a Room</button>
+		<a class="btn btn-warning text-decoration-none" data-route href="/tounament/pong">Join a Tournament</a>
 	</div>
 
 	<div id="join" game-menu>
```

Comment:
An id of join is not ideal, try to be more precise

Reply:
ok